modules/ossm-supported-configurations.adoc

```diff
@@ -9,13 +9,13 @@
 
 The following configurations are supported for the current release of {SMProductName}:
 
-* Red Hat {product-title} version 4.9 or later.
-* Red Hat {product-dedicated} version 4.
+* {product-title} version 4.6 or later.
+* {product-dedicated} version 4.
 * Azure Red Hat OpenShift version 4.
 
 [NOTE]
 ====
-Red Hat OpenShift Online is not supported for {SMProductName}.
+OpenShift Online is not supported for {SMProductName}.
 ====
 
 * This release of {SMProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
```

Comment:
When I read the doc preview of this topic, the logic behind this list seems to "fall apart" here on line 21.  Should we move this line and its two sub-bullets out of this list?
The result would look like this: 
```
This release of {SMProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
* IBM Z is only supported on {product-title} 4.6 and later.
* IBM Power Systems is only supported on {product-title} 4.6 and later.```

Reply:
Same issue in `modules/ossm-supported-configurations-v1x.adoc`